### PR TITLE
Refactored the slack & unsplash settings

### PR DESCRIPTION
**Pull request by allouis**
refs #10318 

- [x] moves the `slack.url` setting into `slack_url`
- [x] moves the `slack.username` setting into `slack_username`
- [x] moves the `unsplash.isActive` setting into `unsplash_is_active`
- [x] updates importer to skip `slack_url`
- [x] updates slack service to use new settings
- [x] updates settings api to handle old settings

### CODE REVIEW

**Comment by allouis** [comment]
https://github.com/TryGhost/Ghost/pull/12050/commits/c414ff581c6f241dce5951d41f21f5c7ea6b999a & https://github.com/TryGhost/Ghost/pull/12050/commits/c414ff581c6f241dce5951d41f21f5c7ea6b999a contain the migrations :relaxed:

File: core/server/data/migrations/versions/3.26/02-refactor-slack-setting.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+module.exports = {
+    config: {
+        transation: true
```
Comment (daniellockyer) [inline core/server/data/migrations/versions/3.26/02-refactor-slack-setting.js:3]: typo :pencil2:

File: core/server/data/migrations/versions/3.26/03-refactor-unsplash-setting.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+module.exports = {
+    config: {
+        transation: true
```
Comment (daniellockyer) [inline core/server/data/migrations/versions/3.26/03-refactor-unsplash-setting.js:3]: typo :pencil2:

**Review by daniellockyer — CHANGES_REQUESTED** [review]

File: core/server/data/migrations/versions/3.26/02-refactor-slack-setting.js (hunk 1 above)
Comment (allouis) [reply core/server/data/migrations/versions/3.26/02-refactor-slack-setting.js]: :grimacing:

File: core/server/data/migrations/versions/3.26/03-refactor-unsplash-setting.js (hunk 2 above)
Comment (allouis) [reply core/server/data/migrations/versions/3.26/03-refactor-unsplash-setting.js]: :grimacing:

**Comment by ErisDS** [comment]
This is missing handling for unsplash in the importer?

**Review by hc0503 — APPROVED** [review]

**Comment by allouis** [comment]
Closing to keep repo clean, will keep the branch if we decide we need this again!

**Pull request closed without merging**